src/main/java/core/basesyntax/strategy/impl/CreditOperationProcessor.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.Map;
+import java.util.Optional;
+
+public class CreditOperationProcessor implements OperationHandler {
```

Comment:
```suggestion
public class CreditOperationHandler implements OperationHandler {
```
the same for other implementation of OperationHandler